action.yml

```diff
@@ -439,7 +426,8 @@ runs:
       if: ${{ env.CA_GH_ACTION_INGRESS_ENABLED == 'true' && inputs.targetPort == '' && env.CA_GH_ACTION_DOCKERFILE_PATH == '' && inputs.appSourcePath != '' && env.CA_GH_ACTION_YAML_PROVIDED != 'true' }}
       shell: bash
       run: |
-        if [[ "${{ env.CA_GH_ACTION_RUNTIME_STACK }}" == "python:"* ]]; then echo "CA_GH_ACTION_TARGET_PORT=80" >> $GITHUB_ENV; else echo "CA_GH_ACTION_TARGET_PORT=8080" >> $GITHUB_ENV; fi
+        CA_GH_ACTION_TARGET_PORT="8080"
+        echo "CA_GH_ACTION_TARGET_PORT=${CA_GH_ACTION_TARGET_PORT}" >> $GITHUB_ENV
```

Comment:
Could you please clarify where the Python exception related to port 80 moved? Did we not need it anymore?

Reply:
OK I knew there was something I forgot to clean-up 😄 The Oryx runtime images we use have a default port that they listen on if one wasn't provided as an environment variable, so if the user didn't provide a port for their Container App, we defaulted to the port that was defined on the platform's corresponding runtime image. Since we can update the port in the builder via the `ORYX_RUNTIME_PORT` environment variable now, we can just set a universal default of 8080 and provide that to this environment variable and not have it matter what platform was used.

Reply:
OK makes sense. Thanks for clarifying!